docs/data/charts/tooltip/ItemTooltipTopElement.js

```diff
@@ -0,0 +1,58 @@
+import * as React from 'react';
+
+import NoSsr from '@mui/material/NoSsr';
+import Popper from '@mui/material/Popper';
+import { useItemTooltip, useMouseTracker } from '@mui/x-charts/ChartsTooltip';
+import { useSvgRef, useXAxis, useXScale, useYScale } from '@mui/x-charts/hooks';
+import { CustomItemTooltipContent } from './CustomItemTooltipContent';
+import { generateVirtualElement } from './generateVirtualElement';
+
+export function ItemTooltipTopElement() {
+  const tooltipData = useItemTooltip();
+  const mousePosition = useMouseTracker();
+  const xAxis = useXAxis(); // Get xAxis config to access its data array.
+  const xScale = useXScale(); // Get the scale which map values to SVG coordinates. Pass the axis id to this hook if you use multiple one.
+  const yScale = useYScale(); // Get the scale which map values to SVG coordinates. Pass the axis id to this hook if you use multiple one.
+  const svgRef = useSvgRef(); // Get the ref of the <svg/> component.
+
+  if (!tooltipData || !mousePosition || !xAxis.data) {
+    // No data to display
+    return null;
+  }
+
+  if (
+    tooltipData.identifier.type !== 'bar' ||
+    tooltipData.identifier.dataIndex === undefined ||
+    tooltipData.value === null
+  ) {
+    // This demo is only about bar charts
+    return null;
+  }
+
+  const xValue = xAxis.data[tooltipData.identifier.dataIndex];
+
+  const svgYPosition = yScale(tooltipData.value) ?? 0 + yScale.step() / 2;
+  const svgXPosition = xScale(xValue) ?? 0;
+
+  const tooltipPosition = {
+    ...mousePosition,
+    x: svgRef.current.getBoundingClientRect().left + svgXPosition,
+    y: svgRef.current.getBoundingClientRect().top + svgYPosition,
+  };
```

Comment:
Would be good to explain what we are doing here 😅 

Eg: 
- why `yScale.step() / 2`
- why we get bounding box?